### DISCUSSION TITLE
Would it be worth proposing to add `simple/codeberg` as popular git icon?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#5887] [Q&A]
I was wondering, if it would be any worth to PR the addition of `simple/codeberg` to the [Repository Icon](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#repository-icon) section, as I feel like it's another popular repository host (Especially given its core values for free and open source projects).

It would be an extremely small addition which is why I want to ask first before wasting everyone's time with a PR that may not be merged.

Tho, I also want to ask, if it wouldn't be also worth to switch out the `fontawesome/brands/...` icons with the `simple/...` variants where available.

Simple icon SVGs seem to be under C0 by default, if no other License is defined, making them a lot more open compared to Fontawesome (CC-BY 4.0 last time I checked). Also, they actively promote their open source nature while FA doesn't (They aren't OS iirc? Outside of having their icons under CC license I mean).
This could promote and encourage open source itself in some way perhaps? Tho that's a personal opinion here, so take it with a grain of salt.

My main point is simply the question I asked first (To add another git repo host icon example), so if that one can be answered, I would be happy.

**Comment by squidfunk** [comment]
We can switch the brand icons from fontawesome to simple icons, but we're not considering adding codeberg.
